### DISCUSSION TITLE
My experience installing the local dev environment

### DISCUSSION THREAD

**Discussion by tm-jc-nacpil** [thinkingmachines/geowrangler#86] [General]
Hi! I was following the local dev installation instructions, and would like to share some feedback. TL;DR It went pretty smoothly naman, and I have some small suggestions for the instructions (in bold)

1. Before running, verified the python version to be `3.9.12`
2. **After forking and cloning the repo, I had to `cd geowrangler` first for the rest of dev set-up instructions to work. Adding instructions for cloning and moving to the forked repo may be helpful**
3. poetry install created a default virtual environment
a. Log: Creating virtualenv geowrangler-hjXmHUg_-py3.9 in /home/jc_tm/.cache/pypoetry/virtualenvs
b. **I had to google how to activate the default environment that it created, so maybe we could add this line to the instructions:  `source {path_to_venv}/bin/activate`**
4. Rest of the installation instructions completed with no issue :smile:
5. After running poetry run jupyter lab it leads me to a jupyter lab window with the directory in the mismong repo
6. END

**Comment by jtmiclat** [comment]
For number 3, you can activate the env by running `poetry shell`
